### PR TITLE
docs: added 3 new sections to FAQ.md

### PR DESCRIPTION
**Pull request by vincentchen137**
Checklist:
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Added 3 new sections to FAQ.md to help both new and experienced OSS contributors. 

“What do I need to know to contribute to the codebase”, seemed like a worthy addition to the page because it lists out all the technologies that this project uses. So, anyone that skipped directly to the FAQ page of the contributor’s documentation will be able to see this important information right away. 

“Can I contribute articles to freeCodeCamp News or videos to freeCodeCamp’s YouTube channel”, also seemed like a good addition to me because it offered another way for potential contributors to get involved if they do not want to contribute directly to the codebase or translate freeCodeCamp’s learning platform. The publication guide and the YouTube channel guide are both hidden in the articles section of freeCodeCamp but are well written and are certainly important enough to be on the FAQ. 

“How can I help review Pull Requests that are ready for quality assurance and approval?”, could be extremely helpful for more experienced but new to the community OSS contributors that want to review Pull Requests for quality assurance.

### CODE REVIEW

File: docs/FAQ.md
Comment (raisedadead) [inline docs/FAQ.md:24]: Thanks for adding these. 

I think instead of the publication style guide which is more formal, we should point them to https://www.freecodecamp.org/news/how-to-write-for-freecodecamp - which is about what it means to be an author and introduces them to the style guide.

Maybe we can put both links here and reword it somehow to let people choose which one they want to hop into.

File: docs/FAQ.md
Comment (raisedadead) [inline docs/FAQ.md:79]: Thanks this is great, I think we should add something about how you can join the moderator's team here too. Because while we encourage reviews, we encourage new contributors to OSS "to be part of the team" and be involved.

There is a moderators handbook that you can use for cross linking.

**Review by raisedadead — COMMENTED** [review]
Just some thoughts, but @freeCodeCamp/dev-team I would love to hear what you think. We do want more folks reviewing code.

File: docs/FAQ.md
Comment (nhcarrigan) [reply docs/FAQ.md:24]: I do like the idea of including both links.

File: docs/FAQ.md
Comment (nhcarrigan) [reply docs/FAQ.md:79]: In my experience, actively encouraging non-maintainer reviews creates a lot of noise. I've seen in other orgs cases where approvals are given without understanding the changes (thus, approvals that should not have been there), or even approvals after a PR is merged.

File: docs/FAQ.md
Comment (ShaunSHamilton) [reply docs/FAQ.md:24]: Personally, I would just update the contribute.freecodecamp.org landing page to have links to:
- How to become an author
- How to contribute to the freeCodeCamp YouTube channel

---

One thing to note: I do not think we are currently able to seek contributions towards YouTube captions.

File: docs/FAQ.md
Comment (ShaunSHamilton) [reply docs/FAQ.md:79]: > add something about how you can join the moderator's team here too.

Well, I am not sure what that is. So, probably a nice idea to add 😅 

I am happy with a short section on helping QA, but suggest even less:
```suggestion
### How can I help review pull requests that are ready for quality assurance?

You can review PRs that need to be quality assured using [this page](https://github.com/freeCodeCamp/freeCodeCamp/pulls?q=is%3Aopen+is%3Apr+-label%3A%22status%3A+blocked%22+-label%3A%22status%3A+merge+conflict%22+status%3Asuccess+draft%3Afalse).
```

Comment (raisedadead) [reply docs/FAQ.md:79]: I think @nhcarrigan is correct about non-maintainer reviews becoming noise. 

That said, I am in favour of coming up with some guidelines on how to contribute in a meaningful way to reviews. A path to join the moderator's team would be helpful IMHO. 

We always need more people on the @freeCodeCamp/moderators team because volunteers are busy, and we need all the help we can get.

Comment (nhcarrigan) [reply docs/FAQ.md:79]: I agree with putting together a path for joining the moderation team.

Comment (raisedadead) [reply docs/FAQ.md:79]: Do you have some recommendations, considering your experience with communities outside of freecodecamp? (we can hash the details on this PR or a dedicated one once we settle on something offline, if you prefer)

Comment (ArielLeslie) [reply docs/FAQ.md:79]: In the past we've had a very "don't call us, we'll call you" approach to becoming a moderator because we want to pull from community leaders. It does seem to me that ideally we would want moderators to have been consistent members of the platform for a certain amount of time. I think that being familiar and comfortable with our community is really important.
On the forum it's pretty easy to look at a history of their interactions with other campers, but I don't know much about what criteria you could use to vet a Discord member.

Comment (thecodingaviator) [reply docs/FAQ.md:79]: Hi, I’ll chime in a bit on this. I think there are discord bots that will measure and track users’ message history and activity. That data can be useful to what we’re trying to achieve

Comment (nhcarrigan) [reply docs/FAQ.md:79]: @raisedadead realistically, the path would primarily reflect how to engage with our platforms in meaningful ways.

As @ArielLeslie mentioned, our process has always been to identify and reach out to active members and invite them to join the team - so I would argue that the best "roadmap" would focus on how to contribute meaningfully on the forum/discord/chat/github, etc.

Comment (raisedadead) [reply docs/FAQ.md:79]: Thanks for those great insights, Ariel and Parth. Appreciate the feedback. 

Parth, we already have a dashboard to get metrics from multiple communities (GitHub, forum, Discord, and more) with our Orbit integration (essentially we have a way to identify folks who are most active and where etc.), that's not what we need though right now.

At the end of the day, we would still follow the "don't call us, we'll call you" approach, that Ariel correctly has pointed out.

The idea I guess from this PR is: 

We want to document, how as a newcomer to freeCodeCamp's community, you can be on that path of establishing your credibility (over a certain period of time). Documenting the process in verbiage that talks about the recommended way to approach being a moderator should point interested folks in the right direction, IMHO.

Suggestions about what we should include in such documentation (for newcomers) would be awesome.

Comment (ArielLeslie) [reply docs/FAQ.md:79]: So maybe more of some suggestions on how best to offer help and leadership? Things like tips on how to answer questions effectively, when to flag posts for moderation, using encouraging/supportive language, cross-cultural communication, avoiding online drama...
Maybe an external News article that could be linked in a 'so you wanna moderate' section?

Comment (nhcarrigan) [reply docs/FAQ.md:79]: > Things like tips on how to answer questions effectively, when to flag posts for moderation, using encouraging/supportive language, cross-cultural communication, avoiding online drama...

I like these ideas and think we would definitely benefit from having them. I'm not sure if a news article is better than having it within the documentation, only because it is a bit harder to edit/maintain a news article (fewer people have access to change the content).

I think we should probably create an issue to discuss + brainstorm content, so we don't bog down this PR too much.

Comment (raisedadead) [reply docs/FAQ.md:79]: > I think we should probably create an issue to discuss + brainstorm content, so we don't bog down this PR too much.

I agree.

@vincentchen137 thanks for the patience, do you mind removing this section while we discuss this in a separate thread. I think the rest of the PR can go in.

Comment (raisedadead) [reply docs/FAQ.md:79]: I created a thread, lets move our discussion there instead.

Comment (vincentchen137) [reply docs/FAQ.md:79]: @raisedadead yes of course! I'll send a commit without that section shortly. 

Sorry if I added this section unnecessarily, I had found your pinned thread with this information on the forums and thought it would be helpful in the FAQ section to entice more experience OSS contributors to join the community. I am glad, however, that it's sparked a conversation on how contributors could get more involved in helping moderators directly and perhaps even becoming one.

File: docs/FAQ.md
Comment (vincentchen137) [reply docs/FAQ.md:24]: @ShaunSHamilton Do you know if we're currently seeking contributions towards creating YouTube videos? If not, I can omit the Youtube part entirely from this PR.

Comment (vincentchen137) [reply docs/FAQ.md:24]: @raisedadead @nhcarrigan Sounds great! I'll include both links.

File: docs/FAQ.md
Comment (raisedadead) [reply docs/FAQ.md]: Thanks @vincentchen137 - we are glad as well we are having this conversation as well. Cheers.

File: docs/FAQ.md
Comment (raisedadead) [reply docs/FAQ.md]: We are creating videos with help from the community but no longer accepting contributions to captions because it's a YouTube limitation - they have since removed the community contribution feature altogether. 

This bit about YouTube can be removed.

Comment (vincentchen137) [reply docs/FAQ.md]: Just removed the bit about captions and translations for YouTube videos in my latest commit :)

File: docs/FAQ.md
Comment (ieahleen) [inline docs/FAQ.md:21]:
```suggestion
If you're interested in writing articles for freeCodeCamp News, please visit this [publication guide](https://www.freecodecamp.org/news/how-to-write-for-freecodecamp/). In addition, please read our [style guide](https://www.freecodecamp.org/news/developer-news-style-guide/) as this will help you write stronger and more effective articles.
```

missing spaces

**Review by naomi-lgbt — DISMISSED** [review]
This is looking good. Thanks for all your work on this!

File: docs/FAQ.md
Comment (vincentchen137) [reply docs/FAQ.md]: Thanks for catching that!

File: docs/FAQ.md
Comment (ShaunSHamilton) [inline docs/FAQ.md:7]:
```suggestion
freeCodeCamp runs on a modern JavaScript stack. If you're interested in contributing to our codebase, you will need some familiarity with JavaScript and some of the technologies we use like Node.js, MongoDB, OAuth 2.0, React, Gatsby, and Webpack.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Final (probably) change request :)

File: docs/FAQ.md
Comment (vincentchen137) [reply docs/FAQ.md:7]: Looks good to me!

**Review by ShaunSHamilton — APPROVED** [review]
Thank you, for taking the time to contribute.

This LGTM 👍

**Review by thecodingaviator — APPROVED** [review]
LGTM!

**Pull request merged**
